analyzers/src/SonarAnalyzer.CSharp/Rules/UnconditionalJumpStatement.cs

```diff
@@ -42,12 +42,12 @@ public sealed class UnconditionalJumpStatement : UnconditionalJumpStatementBase<
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
         protected override GeneratedCodeRecognizer GeneratedCodeRecognizer => CSharpGeneratedCodeRecognizer.Instance;
         protected override ISet<SyntaxKind> LoopStatements { get; } = new HashSet<SyntaxKind>
-        {
-            SyntaxKind.ForEachStatement,
-            SyntaxKind.ForStatement,
-            SyntaxKind.WhileStatement,
-            SyntaxKind.DoStatement
-        };
+            {
```

Comment:
pedantic: I think we don't need to shift this with 4 spaces. The other field you changed is aligned with the visibility modifier.